tests/e2e-pdb/target-allocator/01-install.yaml

```diff
@@ -0,0 +1,149 @@
+# This creates four different deployments for checking different pdb options:
```

Comment:
[I did the same for the collector pdb](https://github.com/open-telemetry/opentelemetry-operator/blob/main/tests/e2e-pdb/pdb/01-install.yaml). Should I remove them there too? I added all the cases because it allows IntOrString as type (and it's quite "magic" for me).
It doesn't have any extra cost and covers it but I can remove them, keeping just one, as you prefer

Reply:
Those can be removed as well